test/cli/samples/warn-unknown-options/rollup.config.js

```diff
@@ -0,0 +1,22 @@
+var replace = require('rollup-plugin-replace');
+var assert = require('assert');
+
+let warnings = 0;
+
+module.exports = commands => ({
+	input: 'main.js',
+	plugins: [
+		{
+			ongenerate() {
+				assert.equal(warnings, 1);
+			}
+		},
+		replace({ 'ANSWER': commands.configAnswer }),
+	],
+	onwarn(warning) {
+		warnings++;
+		assert.equal(warning.code, 'UNKNOWN_OPTION');
+		assert.equal(warning.message,
+			'Unknown CLI option: unknownOption. Allowed options: acorn, acornInjectPlugins, amd, banner, c, cache, config, context, d, dir, e, entry, environment, experimentalCodeSplitting, experimentalDynamicImport, experimentalPreserveModules, exports, extend, external, f, file, footer, format, freeze, g, globals, h, i, indent, input, interop, intro, l, legacy, m, moduleContext, n, name, namespaceToStringTag, noConflict, o, onwarn, outro, paths, perf, plugins, preferConst, preserveSymlinks, silent, sourcemap, sourcemapFile, strict, treeshake, v, w, watch');
```

Comment:
The main reason I say this is that `Allowed options` here may not immediately be clear to users that these are flags without using the word `flag` (at least it was to me - I would have interpreted these as commands `rollup c` etc instead of `rollup -c`...).